x-pack/test/functional/services/ml/deploy_models_flyout.ts

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { IngestInferenceProcessor } from '@elastic/elasticsearch/lib/api/types';
+import { ProvidedType } from '@kbn/test';
+
+import type { FtrProviderContext } from '../../ftr_provider_context';
+import type { MlCommonUI } from './common_ui';
+
+export interface TrainedModelRowData {
+  id: string;
+  description: string;
+  modelTypes: string[];
+}
+
+export type MlDeployTrainedModelsFlyout = ProvidedType<typeof DeployTrainedModelsFlyoutProvider>;
+
+export function DeployTrainedModelsFlyoutProvider(
```

Comment:
wdyt?
```suggestion
export function DeployDFAModelsFlyoutProvider(
```

Reply:
Updated in 001a8301d6190c0a994c65cff3a18105aeda53b4